rmgpy/rmg/model.py

```diff
@@ -1429,7 +1429,16 @@ def addSeedMechanismToCore(self, seedMechanism, react=False):
                  reversible=rxn.reversible
                  )
             r, isNew = self.makeNewReaction(rxn) # updates self.newSpeciesList and self.newReactionlist
-            if not isNew: logging.info("This library reaction was not new: {0}".format(rxn))
+            if not isNew:
```

Comment:
Why do we need this processing changes for seeds and reaction libraries?

Reply:
To convert each new reaction from libraries/seeds which has an Arrhenius-type kinetics (and is flagged as high-P elementary) into a PDep-type kinetics
(`processNewReactions` calls `rxn.generate_high_p_limit_kinetics()`)